libraries/ms-common/src/x86/gpio.c

```diff
@@ -29,10 +29,12 @@ static uint32_t prv_get_index(const GPIOAddress *address) {
 }
 
 StatusCode gpio_init(void) {
-  GPIOSettings default_settings = { .direction = GPIO_DIR_IN,
-                                    .state = GPIO_STATE_LOW,
-                                    .resistor = GPIO_RES_NONE,
-                                    .alt_function = GPIO_ALTFN_NONE };
+  GPIOSettings default_settings = {
+    .direction = GPIO_DIR_IN,
```

Comment:
Interesting. I think it would be helpful to have this link and a brief explanation of the problem in the clang format file so people know why we have specific options set.

I think using this and using `//` to force line breaks manually is probably the best thing to do right now. Ultimately, this is just a tool to help make our lives easier and we can always change it if we need to. :P